src/services/collections.js

```diff
@@ -1,19 +1,32 @@
-import config from 'config';
-
 import fetch from '../utils/fetch.js';
 
-export const fetchCollections = async () => {
-  const collections = await fetch(config.get('collectionsUrl'));
+export const fetchCollections = async collectionsConfig => {
+  let collections = [];
+
+  for (const item of collectionsConfig) {
+    if (typeof item === 'string') {
+      /* eslint-disable no-await-in-loop */
+      const fetchedCollections = await fetch(item); // Use `await` to ensure sequential fetching of collections and preserve the order in the resulting collections array
```

Comment:
Using `Promise.allSettled(collectionsConfig.map())` would be more efficient and would not contradict the linter (and from my point of view would be more readable / elegant).

Reply:
But it will not preserve the order as all fetch will be made in parallel. 
That why I voluntary use a native for loop with `await` to ensure sequentiality.
So, I guess my comment is not clear enough…

Reply:
The order would be preserved with a `map` and `allSettled`: all promises would run in parallel, and you would continue only after all have settled.